daemon/logger/loggerutils/file_windows.go

```diff
@@ -0,0 +1,224 @@
+package loggerutils
+
+import (
+	"os"
+	"syscall"
+	"unsafe"
+
+	"github.com/pkg/errors"
+)
+
+func openFile(name string, flag int, perm os.FileMode) (*os.File, error) {
```

Comment:
Should we move this to pkg/system? In that package we also have some other windows-specific utilities (some of which) can be used as drop-in replacements for their default go implementations, e.g. https://github.com/moby/moby/blob/e554ab558985b686c9c3427275a5e016aa1cdb76/pkg/system/filesys_windows.go#L29-L33

Reply:
> I'm intending this as a targeted workaround for a very specific and real issue rather than a generic solution.

Gotcha. I was thinking if other places would also benefit from this, having it in that package would make it a slightly bit more reusable; not a blocker though

> It would be nice if go picked up some change to support this option

Yes, definitely